inscriptions/ISic003203.xml

```diff
@@ -189,14 +194,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#FP">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <persName type="attested"><name type="nomen"><supplied reason="lost">Ma</supplied>nliae</name></persName> <persName type="attested"><name type="praenomen"><expan><abbr>T</abbr><ex>iti</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>iliae</ex></expan></w>
+                    <lb n="2"/><supplied reason="lost"><w>et</w></supplied> <persName type="attested"><supplied reason="lost"><name type="praenomen">Marci</name></supplied> <name type="nomen"><supplied reason="lost">Ro</supplied>sci</name> <persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>ili</ex></expan></w> <orgName type="tribus" nymRef="#Quirina"><expan><abbr>Qui</abbr><ex>rina</ex></expan> <supplied reason="subaudible"><w lemma="tribus">tribu</w></supplied></orgName> 
+                    <lb n="3"/><name type="cognomen">Cironni</name></persName> <w>uxori</w> <supplied reason="omitted"><w>piissimae</w></supplied> 
```

Comment:
delete the `<supplied reason="omitted"><w>piissimae</w></supplied>` there is no reason to include this that I can see, the right margin is clearly intact and there's no basis for it
![DSC_5748](https://user-images.githubusercontent.com/25080386/133067370-35af33d8-0d89-4db1-ac95-cb88848b4867.JPG)